wolfcrypt/src/asn.c

```diff
@@ -13727,6 +13727,11 @@ int wc_ValidateDate(const byte* date, byte format, int dateType)
     (void)tmpTime;
 
     ltime = wc_Time(0);
+    if (ltime < 0){
+        /* This could be because 32-bit and the year is >=2038 */
```

Comment:
How about: `A negative response here could be due to a 32-bit time_t where the year is 2038 or later.`